solutions/secure/README.md

```diff
@@ -0,0 +1,12 @@
+# IBM Cloud Databases for Elasticsearch
+
+This architecture creates an IBM Elastic Search instance in IBM Cloud environment and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Elastic search instance on IBM Cloud
+- Creation of CBR rules
+- Supports autoscaling
```

Comment:
Creates context based restriction(CBR) rules for.....
Enables/disables autoscaling

Can we add anything for KMS encryption?

Reply:
added